### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, 1:30 pm, Friday Feb 11, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2940] [Meeting Agendas]
The agenda for this coming week is as follows:

1. Jason update
2. Peter update
3. Dong update
    - #2938
4. Cold cases:
a. #2194 (Don)
b. #412 - At the very least, if we choose to not use pandoc, we might be able to learn from some of its [inner mechanics](https://hackage.haskell.org/package/pandoc-types-1.22.1/docs/src/Text.Pandoc.Definition.html#Pandoc)  (Jason)
d. #1715 (Peter)
5. Questions/Comments/Concerns/Discussion

**Comment by cd155** [comment]
For game design topic we mentioned today

1. This is a link for a runnable game in Windows and Macs
https://drive.google.com/drive/folders/1_ixxwUMgE_83385RSp2vooReIQ0NvpYH?usp=sharing

2. The design document about how to play the game and my rationale.
https://github.com/cd155/Game_Jam_Assignments/tree/main/Jumping

**Comment by smiths** [comment]
Great game @cd155.  I'll admit that I died the first time, but I did much better after I read the rules.  :smile:  I thought I would play until I died a second time, but once you figure out what is expected, it is hard to lose.  :smile:  I agree that it would be a great game for children.

**Reply by cd155** [reply]
Since we only have two weeks to implement the game, I only make level one available. There could be some improvements. For example, currently, the fruits are randomly generated in four equally divided channels. If we could reduce the randomness, and have more control over the fruit movement, it can be more flexible to make the game easier or harder.

Thanks for your time to play the game. It's very fun to start making games. :D
